build.gradle

```diff
@@ -86,19 +86,10 @@ task check {
     dependsOn checkExamples
 }
 
-task integrationTestsConnectedCheck(type:GradleBuild) {
```

Comment:
Yeah, i think it is good to still have this since lots of stuff can only be tested through this way. But on current source tree, this is only used for optional API transformer.